agent/storage/api/storage.go

```diff
@@ -19,6 +22,15 @@ type AgentStorage interface {
 	DIDStorage() DIDStorage
 	ConnectionStorage() ConnectionStorage
 	CredentialStorage() CredentialStorage
+
+	OurPackager() Packager
```

Comment:
I wouldn't add packager here: I would try to avoid transforming storage to indy-wallet-like god-service. We should probably discuss online and try to figure out together the best structure for this.

Reply:
I strongly disagree with that, but maybe it's better to have the discussion online. We really need to have polymorphic root entities and core concepts or we cannot handle everything nicely.